src/frontend/src/app/core/layout/secondary-navbar/secondary-navbar.component.ts

```diff
@@ -73,16 +75,59 @@ import { DropdownComponent, TooltipDirective } from '~core/ui';
       .profile-menu__item-icon {
         margin-right: size(2);
       }
+
+      .secondary-navbar__link {
+        color: $white;
+
+        @include no-underline;
+        @include py(2);
+        @include mx(4);
+        border-bottom: 3px solid transparent;
```

Comment:
There's two variable for borders: `$border-size` and `$border-size-large`. Can either of them work?